test/Microsoft.IdentityModel.Tokens.Saml.Tests/SamlSecurityTokenHandlerTests.cs

```diff
@@ -80,6 +79,8 @@ public void CanReadToken(SamlTheoryData theoryData)
             {
                 theoryData.ExpectedException.ProcessException(ex);
```

Comment:
You need to pass context to: ProcessNoException AND ProcessException.
ProcessNoException(context), ProcessException(ex, context)

Reply:
Fixed